clean-code-challanges/src/main/java/Anagram.java

```diff
@@ -1,17 +1,35 @@
-import java.util.List;
+import java.util.*;
 
 /**
  * Given a word and a list of possible anagrams, select the correct sublist.
  *
  * Given "listen" and a list of candidates like "enlists" "google" "inlets" "banana" the program should return a list containing "inlets".
  */
 public class Anagram {
+  private final String word;
 
   public Anagram(String word) {
+    this.word = word.toLowerCase();
 
   }
 
   public List<String> match(List<String> candidates) {
-    return null;
+    String sortedWord = stringSortAlphabet(word);
+    List<String> result = new ArrayList<String>();
+    for (String text:candidates) {
+      if (!word.equals(text.toLowerCase())){
```

Comment:
ich würde text hier anders bennenen (persönlich habe ich candidate of candidates genutzt)

Reply:
Danke, werde ich in die nächste Änderung einfliessen lassen.